### PR TITLE
Update README to use the restricted dev template for Minishift.

### PR DESCRIPTION
**Pull request by rhuss**
This seems to be the most popular installation variant these days.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,21 +160,23 @@ With Minishift you can easily try out Syndesis. The only prerequisite is that yo
...
+Here are step-by-step the installation instructions for setting up a Minishift installation in an restricted OpenShift environment: 
+
 ```bash
 # Fire up minishift if not alread running. 
 # 4 MB of memories are recommended
 minishift start --memory 4192
 
 # Login as admin
-oc login -u system:admin
+oc login -u developer -p developer
```
Comment (jimmidyson) [inline README.md]: Doesn't `minishift start` already do this? Can't remember but thought this was redundant.

**Comment by jimmidyson** [comment]
@rhuss I think you mistakenly pushed to a branch in the origin repo, just a heads up for next time please push to your fork (I know you know that, but just had to say it!).

File: README.md (hunk 1 above)
Comment (rhuss) [reply README.md]: yeah, good point let me check.

**Comment by rhuss** [comment]
Actually, I used the Github Web UI to edit and instead of committing it directly (which I do for smaller typos, shame on me), I went for a PR. I will delete the branch asap after the merge. 

For documentation, I find it very convenient to fix it directly in the browser as the editor is quite matured these days. Unfortunately, my master branch is not really up-to-date all the time, so I took the lazy route (which hopefully is not that disturbing if I delete the branch. which I forget from time to time).

**Comment by jimmidyson** [comment]
Ah yeah that is nice and easy.

**Review by jimmidyson — APPROVED** [review]

**Comment by jimmidyson** [comment]
Done and branch deleted.

**Pull request merged**